projects/auth0-angular/src/lib/auth.guard.ts

```diff
@@ -42,7 +42,9 @@ export class AuthGuard implements CanActivate, CanLoad, CanActivateChild {
     return this.auth.isAuthenticated$.pipe(
       tap((loggedIn) => {
         if (!loggedIn) {
-          this.auth.loginWithRedirect({ appState: { target: state.url } });
+          return this.auth.loginWithRedirect({
```

Comment:
We do not need to add a return here. In fact, this is a tap and shouldnt need the `return of(true)` neither. However, this is a different problem and we will tackle this in another PR. So I would keep this file untouched in this PR.

Reply:
`noImplicitReturns` required that change.
I would remove `return of(true)` or wait that PR

Reply:
We can keep the return here. It has no side-effects anyway.